AddressBook_/AddressBook_/MemberTableViewCell.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  MemberTableViewCell.swift
+//  AddressBook_
+//
+//  Created by 이중엽 on 2023/03/10.
+//
+
+import UIKit
+import SnapKit
+
+class MemberTableViewCell: UITableViewCell {
+
+    let favoriteButton          = UIButton()
+    let memberImageView         = UIImageView()
+    
+    let nameLabel               = UILabel()
+    let phoneNumberLabel        = UILabel()
+    let memoLabel               = UILabel()
+    
+    let nameDataLabel           = UILabel()
+    let phoneNumberDataLabel    = UILabel()
+    let memoCheckButton         = UIButton()
+    
+    // ********************************
+    // MARK: - initialSetup
+    // ********************************
+    func initialSetup(index: Int) {
+        // 기본 배경 색상
+        contentView.backgroundColor = .white
+        
+        // 즐겨찾기 버튼
+        let isFavorite = MemberManger.shared.memberIsFovorite(index: index)
+        favoriteButton.frame.size = CGSize(width: 30, height: 30)
+        if isFavorite {
+            favoriteButton.setImage(UIImage(systemName: "star.fill"), for: .normal)
+        } else {
+            favoriteButton.setImage(UIImage(systemName: "star"), for: .normal)
+        }
+        
+        // 멤버 이미지 뷰
+        // let memberImage = MemberManger.shared.memberImage(index: index)
```

Comment:
혹시 사용하지 않는 주석인 경우는 삭제, 
향후 사용예정의 코드의 경우는 위에 명시해주세요.
```suggestion
       // TODO: 향후 000 구현할 때 사용 예정.
        // let memberImage = MemberManger.shared.memberImage(index: index)
```